### ISSUE TITLE
Basic auth

### ISSUE THREAD

**Issue by SAOPP** [erickok/transdroid#615]
Hi!

It's possible to implement detection of basic authentication of the host instead of the default app login/pass? For example, when ya turn off app's authentication and turn on basic authentication on reverse proxie for app's host.

Thanks.

**Comment by erickok** [comment]
Most clients use basic auth for authentication. Which are you using?

**Comment by erickok** [comment]
You can set the port number in Advanced Settings. For example 443 for https.

**Comment by SAOPP** [comment]
Yes, I has reconfigure my connection, but as I can see app is can't normal connection establishing, 'coz time to time I got error communication issue, I think it at app side 'coz nzb360 client work fine with my host.

**Comment by erickok** [comment]
It's difficult to say without any details really. Perhaps you also need to set a custom 'folder' (path)?

**Comment by SAOPP** [comment]
I agree, nope, the default folder location is used, app working geate with native basic auth of Transmission side as I remeber that.
